DemoApp/Sources/Views/Controls/DemoControls.swift

```diff
@@ -0,0 +1,359 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import StreamVideo
+import StreamVideoSwiftUI
+import struct StreamChatSwiftUI.ChatChannelView
+import SwiftUI
+import class StreamChat.ChatChannelController
+
+struct AppControlsWithChat: View {
+
+    @Injected(\.streamVideo) var streamVideo
+    @Injected(\.images) var images
+    @Injected(\.colors) var colors
+
+    @Environment(\.chatVideoViewModel) var chatViewModel
+    private var canOpenChat: Bool
+
+    private let size: CGFloat = 50
+
+    @ObservedObject var reactionsHelper = AppState.shared.reactionsHelper
+    @ObservedObject var viewModel: CallViewModel
+    @State private var isChatVisible = false
+
+    init(viewModel: CallViewModel, canOpenChat: Bool = true) {
+        self.viewModel = viewModel
+        self.canOpenChat = canOpenChat
+    }
+
+    var body: some View {
+        VStack {
+            HStack(alignment: .top) {
+                Group {
+                    Spacer()
+
```

Comment:
Feels like too much extra spacing. Off topic, we need to define code conventions for SwiftUI.

Reply:
I did change that a little bit